lang/java/reef-runtime-hdinsight/src/main/java/org/apache/reef/runtime/hdinsight/cli/HDICLI.java

```diff
@@ -104,7 +104,7 @@ public void run(final String[] args) throws Exception {
    *
    * @param applicationId
    */
-  private void kill(final String applicationId) {
+  private void kill(final String applicationId) throws IOException {
```

Comment:
Please document the newly introduced exception via an `@throws` javadoc comment.